scripts/metrics/metrics.py

```diff
@@ -136,6 +136,7 @@
     if (type_ == "embed"):
         n_hvgs = None
         embed = "X_emb"
+        precompute_pca = False
```

Comment:
this is from iso label fix? I guess you merged master into here?

Reply:
Yes, I could remove that to reduce runtime slightly

Reply:
It's not merged from scib master though, I added it here for testing

Reply:
This is part of the iso label fix though, no? I mean... I think it's correct to do this. But not initially the remit of this PR. So just wondering if you also have this in scIB at the moment...

Reply:
I just double-checked and the results don't seem to change if I precompute PCA. Will remove it then

Reply:
this is what caused us to miss the iso label issue (as there was a PCA fallback). So I would keep this in here. I just wanted to make sure whether this code was already in scIB as well (before file moving).

Reply:
The variable is not set to False for embed input in scib master

Reply:
okay, then nvm. mergeable :)